src/intl/en/learn-quizzes.json

```
@@ -613,5 +613,50 @@
+  "smart-contracts-1-prompt": "How are smart contracts characterized?",
+  "smart-contracts-1-a-label": "Smart contracts are just like real (paper) contracts, but stored digitally on the blockchain to save the content securely.",
+  "smart-contracts-1-a-explanation": "Smart contracts use a similar logic to traditional contracts, but otherwise have little in common.",
+  "smart-contracts-1-b-label": "They are linked to an AI system, which allows them to make decisions that trigger blockchain transactions.",
+  "smart-contracts-1-b-explanation": "Smart contracts can actually execute transactions automatically, but only according to an ‘if this then that’ logic that is prescribed in the code. They do not use AI.",
+  "smart-contracts-1-c-label": "They are programs stored on the blockchain that follow an 'if this then that' logic, and are guaranteed to execute according to the rules defined by the code.",
+  "smart-contracts-1-c-explanation": "A smart contract is a piece of code that can be executed automatically according to an 'if this then that' logic.",
+  "smart-contracts-1-d-label": "They are the rules behind the Ethereum Blockchain, developed together with lawyers to ensure legal compliance.",
+  "smart-contracts-1-d-explanation": "Smart contracts are pieces of code that can be created by developers and deployed on a blockchain.",
+  "smart-contracts-2-prompt": "What is a metaphor that most closely characterizes the functioning of smart contracts?",
+  "smart-contracts-2-a-label": "A bank",
+  "smart-contracts-2-a-explanation": "Banks require manual intervention and are structured as hierarchical entities, whereas a smart contract automatically executes without intervention.",
+  "smart-contracts-2-b-label": "A digital vending machine",
+  "smart-contracts-2-b-explanation": "The vending machine will only dispense your desired product after all requirements are met - specific inputs guarantee predetermined outputs. This is similar to the logic of smart contracts.",
+  "smart-contracts-2-c-label": "A calculator",
+  "smart-contracts-2-c-explanation": "smart contract code can be used for calculations, but is not limited to that. Rather, smart contracts are blockchain-based programs that follow an 'if this then that' logic.",
+  "smart-contracts-2-d-label": "A website",
+  "smart-contracts-2-d-explanation": "A website is the frontend which captures user directions. A smart contract is the backend logic where these directions are executed, and the result may be returned.",
+  "smart-contracts-3-prompt": "What is not one of the main characteristics of a smart contract?",
+  "smart-contracts-3-a-label": "Automatic execution",
+  "smart-contracts-3-a-explanation": "The main benefit of a smart contract is that it deterministically executes unambiguous code when certain conditions are met.",
```

> when certain conditions are met

A subtle but import point is how these still need to be poked.

I.e. a DAO vote goes up, people vote and it passes... to actually execute the results there needs to be some interaction by an account that can sign a message to authenticate and provide ETH for gas.... it doesn't just fire because the timer ended. Does that make sense?

Consider:

```suggestion
  "smart-contracts-3-a-explanation": "The main benefit of a smart contract is that it deterministically executes unambiguous code, without human interpretation or bias.",
```